### PR TITLE
FIX: ensures bulk-invite is working properly

### PR DESCRIPTION
**Pull request by jjaffeux**
There was two bugs:
- naming of properties were incorrect s/is_private/isPrivate and s/is_public/isPublic
- a previous refactoring has used `=` for setter when it's an `EmberObject` and `set` should be used

This commit also adds a spec and page objects to ensure this modal is working as expected.

### CODE REVIEW

File: spec/system/post_event_spec.rb [hunk 1]
```diff
@@ -92,4 +94,29 @@
...
+    fab!(:invitable_user_2) { Fabricate(:user) }
+
+    it "can invite users to an event" do
+      visit(post.topic.url)
+
+      post_event_page.open_bulk_invite_modal
+      bulk_invite_modal_page.set_invitee_at_row(invitable_user_1.username, "going", 1)
+      bulk_invite_modal_page.add_invitee
+      bulk_invite_modal_page.set_invitee_at_row(invitable_user_2.username, "not_going", 2)
+      bulk_invite_modal_page.send_invites
```
Comment (Grubba27) [inline spec/system/post_event_spec.rb:117]: would something like this work?
```suggestion
      bulk_invite_modal_page
        .set_invitee_at_row(invitable_user_1.username, "going", 1)
        .add_invitee
        .set_invitee_at_row(invitable_user_2.username, "not_going", 2)
        .send_invites
```

Comment (jjaffeux) [reply spec/system/post_event_spec.rb:117]: yes but Im mixed on this being really better

Comment (jjaffeux) [reply spec/system/post_event_spec.rb]: Ok I did, because I don't know... let's say it's better :+1:

**Review by ZogStriP — APPROVED** [review]

File: spec/system/post_event_spec.rb (hunk 1 above)
Comment (ZogStriP) [reply spec/system/post_event_spec.rb]: I ❤️ the chained version slightly more than the procedural one 👍

Comment (Grubba27) [reply spec/system/post_event_spec.rb]: I feel that has less _noise_

**Pull request merged**